tests/apps/search/test_indexers_courses.py

```diff
@@ -747,10 +825,20 @@ def test_indexers_courses_build_es_query_combined_search(self):
         request = SimpleNamespace(
             query_params=QueryDict(
                 query_string="categories=42&categories=84&query=these%20phrase%20terms&limit=2&"
-                + "language=fr&"
+                + "languages=fr&availability=current&"
                 + "start={start}&end={end}".format(start=start, end=end)
             )
         )
+
+        # Search fragments that are repeated in the query
```

Comment:
this is fine

![iu](https://user-images.githubusercontent.com/1932937/52486888-9da20d80-2bbc-11e9-9f1b-bfad3f82f23f.gif)